system_metrics_collector/test/topic_statistics_collector/test_received_message_period.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <string>
+#include <thread>
+
+#include "../../src/moving_average_statistics/types.hpp"
```

Comment:
relative imports, across the review - let's resolve it

Reply:
Resolved most in this PR (https://github.com/ros-tooling/system_metrics_collector/pull/95) and once that's pushed will rebase and resolve here.